### DISCUSSION TITLE
Upload files and return files in the conversation

### DISCUSSION THREAD

**Discussion by XunDong-Shi** [langgenius/dify#6781] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

In the context of intelligent office, it is often necessary to upload files in PDF/Excel/Word format, process them through AI workflow, and then return them to the user side. However, this feature is currently not available in DIFY. This is a common product requirement, and it is recommended to consider it in product planning

**Comment by Shivak11** [comment]
Second that. We have really been closely watching Dify and want to buy the team edition but this requirement is keeping us at bay. We'd subscribe the day this functionality goes live. Without the ability to upload files, things like data analysis are essentially rendered useless. Ability to upload pdf's and simpler external data connection is really what we need.

**Reply by crazywoola** [reply]
Already started working on it. It will be released in next month.

**Reply by JuHyung-Son** [reply]
does this released?

**Comment by carloscesartc** [comment]
The addition of native support for PDF upload and analysis would make Dify a more robust and versatile solution. This would meet the demand of many companies and industries that need to process and analyze data directly from PDF documents, enabling AI agents to interact with real files and provide more precise and informed responses. The PDF upload and analysis feature would increase Dify's adoption and offer a key competitive advantage.

Thank you for your consideration!
